src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1673,7 +1603,7 @@ fromCommandLine(
     Logs& logs)
 {
     auto const result =
-        rpcClient(vCmd, config, logs, RPC::apiMaximumSupportedVersion);
+        rpcClient(vCmd, config, logs, RPC::apiMinimumSupportedVersion);
```

Comment:
oh I like this idea. For now this will be explicitly 1 because we do not want to push forward too fast (in this release), but to be updated in a separate PR soon after